### PR TITLE
Minor UI changes around crediting an account

### PR DESCRIPTION
**Pull request by brian-weir**

### CODE REVIEW

File: ui/src/pages/page/PageSection.scss [hunk 1]
```diff
@@ -20,7 +20,7 @@
       align-items: flex-start;
       justify-content: space-between;
 
-      :first-child {
+      > :first-child {
```
Comment (Sarahb-da) [inline ui/src/pages/page/PageSection.scss:23]: nice!

File: ui/src/pages/custody/Account.tsx [hunk 2]
```diff
@@ -221,7 +209,7 @@ const AccountComponent: React.FC<RouteComponentProps & ServicePageProps<Service>
     <>
       <BackButton prevPageLabel="Wallet" prevPagePath={paths.app.custody.assets} />
       <InputDialog {...transferDialogProps} isModal />
-      <InputDialog {...creditDialogProps} isModal />
+      {/*<InputDialog {...creditDialogProps} isModal />*/}
```
Comment (Sarahb-da) [inline ui/src/pages/custody/Account.tsx:212]: did u mean to leave this commented out?

**Review by Sarahb-da — APPROVED** [review]
one comment..

**Review by alexgraham-da — APPROVED** [review]
looks good to me 👍

**Review by alexmatson-da — APPROVED** [review]

File: ui/src/pages/custody/Account.tsx (hunk 2 above)
Comment (brian-weir) [reply ui/src/pages/custody/Account.tsx:212]: No I didn't mean to leave this in! Thanks for pointing this out!! 👀

File: ui/src/pages/page/PageSection.scss (hunk 1 above)
Comment (brian-weir) [reply ui/src/pages/page/PageSection.scss:23]: I'm no expert in CSS but this took me a rather embarrassing long time to figure out...

**Pull request merged**